test/board_test.exs

```diff
@@ -124,4 +124,53 @@ defmodule BoardTest do
       assert Board.game_status(board) == :tied
     end
   end
+
+  describe "move_status/1" do
+    test "determines if the input is out of the bounds of the 3x3 board on the upper bounds" do
```

Comment:
I wonder if we can make the test description more explicit, for example:
```ex
describe "move_status/1" do
  test "indicates a move is out of bound" do ...
  test "indicates a move is valid" do ...
```
I like to think of tests as a `README` for the module under test where I can get a high level understanding of what it does without having to dig through the actual code. This also makes it easier to pinpoint where in the code we need to change if one of the tests fails.

Reply:
That is a great point, "determines if", kinda brings up some questions/doubt about the result of the test/what it is testing, but "indicates *does this*" tells someone exactly what the test is doing and what its result should be, I like that alot! Thank you 😄

Reply:
Yea a lot of times I like to mirror the description around the input and output and so when you do TDD it forces you to think about the boundaries: what input and output am I expecting so you can be even more specific like:
```ex
test "returns X given Y" do
```